generate_byte_code.py

```diff
@@ -0,0 +1,22 @@
+import click
+
+from solc import compile_source
+
+
+def convert_to_byte_code(file):
+    path_to_file = open(file, mode='r')
+    compiled_sol = compile_source(path_to_file.read())
+    contract_interface = compiled_sol['<stdin>:{}'.format(str(file.split('.sol')[0]))]
+    print(contract_interface['bin'])
+
+
+@click.command()
+@click.option('--type', help="format to convert")
+@click.option('--file', help="file to convert")
```

Comment:
- [ ] Rename `--file` to `-s` `--solidity-file`
- [ ] Change help text to Uppercase : ) `Path to solidity file`
- [ ] Add `-c` `--contract` param, this parameter define which contract bytecode we need (if empty, provide dict with all contracts and bytecodes)

```
{
    'ContractName': {
        'bytecode': '0x..',
        'abi': '..'
    },
    ContractName2': {
        'bytecode': '0x..',
        'abi': '..'
    }
}